### ISSUE TITLE
Fish 3.2.2 update issue

### ISSUE THREAD

**Issue by nicolaturcato** [FabioAntunes/fish-nvm#57]
Hi there!
Since last update of Fish to v3.2.2 this error appears:

``` Could not send child 19436, 'uname' in job 1, 'uname -s' from group -1 to group 19436
setpgid: No such process
test: Missing argument at index 3
= Darwin
         ^
~/.local/share/omf/pkg/fish-nvm/functions/__nvm_run.fish (line 9):
  if test (uname -s) = 'Darwin'; and string match -q "*versions/node/*/bin" $PATH
     ^
in function '__nvm_run' with arguments 'yarn proxy'
	called on line 2 of file ~/.local/share/omf/pkg/fish-nvm/functions/yarn.fish
in function 'yarn' with arguments 'proxy'
Could not send child 19437, 'cut' in job 1, 'status stack-trace | grep called | cut -d " " -f 7' from group -1 to group 1502
setpgid: No such process
Could not send child 19439, 'which' in job 1, 'which $argv[1]' from group -1 to group 19439
setpgid: No such process
test: Missing argument at index 3
 !=
    ^
Standard input (line 21):
    if type -fq $argv[1]; and test "$stack[1]" != (which $argv[1])
                              ^
in function 'run_command' with arguments 'yarn proxy'
	called on line 68 of file ~/.local/share/omf/pkg/fish-nvm/functions/__nvm_run.fish
in function '__nvm_run' with arguments 'yarn proxy'
	called on line 2 of file ~/.local/share/omf/pkg/fish-nvm/functions/yarn.fish
in function 'yarn' with arguments 'proxy'
Fish nvm: 'yarn' is currently not installed, try running npm i -g yarn

**Comment by FabioAntunes** [comment]
Thanks I will take a look asap

**Comment by nicolaturcato** [comment]
@FabioAntunes thanks Fabio!
Restarting iTerm made it work again. But would be nice if you can have a look at it.

**Comment by ColCh** [comment]
I have fish issue on 3.2.2 too :( just like #54 

<details>
<summary>Details</summary>

<pre>
~/D/Workspace $ nvm use v14                                             16:12:12
Now using node v14.17.0 (npm v6.14.13)
~/D/Workspace $ node                                                    16:12:14
type: Invalid combination of options
Fish nvm: 'node' is currently not installed, try running npm i -g node
~/D/Workspace $ which node                                              16:12:16
/usr/local/opt/nvm/versions/node/v14.17.0/bin/node
~/D/Workspace $ fish --version                                          16:12:18
fish, version 3.2.2
~/D/Workspace $ cd ~/.local/share/omf/pkg/fish-nvm/ && git show HEAD    16:12:26
commit 12938fe (HEAD -> master, tag: 1.4.2, origin/master, origin/HEAD)
Author: Nathan Gilbert <546222+nathan-gilbert@users.noreply.github.com>
Date:   Wed Mar 3 09:51:33 2021 -0700

    fix for fish 3.2.0 (#54)

────────────────────────────────────────────────────────────────────────────────
modified: functions/__nvm_run.fish
────────────────────────────────────────────────────────────────────────────────
@ functions/__nvm_run.fish:21 @ function __nvm_run
    set stack (status stack-trace | grep called | cut -d " " -f 7)
    set count (count $argv)

    if type -fqP $argv[1]; and test "$stack[1]" != (which $argv[1])
    if type -fq $argv[1]; and test "$stack[1]" != (which $argv[1])
      set count (count $argv)
      if test "$count" -ge 2
        set args $argv[2..-1]
</pre>

</details>

**Comment by nicolaturcato** [comment]
All good.
